source/system-administrators/upgrade/upgrading-to-craftercms-3-0.rst

```diff
@@ -133,6 +131,34 @@ before site creation, just before entering the Studio credentials):
 #. Change the date pattern from ``MM/dd/yyyy HH:mm:ss`` to ``yyyy-MM-dd'T'HH:mm:ss.SSSX``, when parsing a ``_dt`` field extracted from the content model
    (make sure ``<disableFullModelTypeConversion>`` is set as true, which the script should have done automatically).
 
+After you've made any necessary changes, commit them.
+
+^^^^^^^^^^^^^^^^^^^^^
+Run the Import Script
+^^^^^^^^^^^^^^^^^^^^^
+
+This is the last step of the migration/import, and basically involves importing the migrated site into Studio. Before continuing, make sure:
+
+  - Studio 3.0.x is running.
+  - There's no site in Studio with the same name as the one you're migrating.
+
+Run the ``import.sh``, which is located under ``crafter-authoring/bin/migration`` directory. The script takes 1 parameter, the name of the site,
+which should be the same one that you used on the migration process.
+
```

Comment:
It would be nice to show a sample of running the script here? Something like:

.. code-block:: sh
    ./import.sh siteNameFromMigration